Xwt.Mac/Xwt.Mac/TableViewBackend.cs

```diff
@@ -41,6 +42,13 @@ public abstract class TableViewBackend<T,S>: ViewBackend<NSScrollView,S>, ITable
 		ScrollView scroll;
 		NSObject selChangeObserver;
 		NormalClipView clipView;
+
+		Dictionary<CellView,CellInfo> cellViews = new Dictionary<CellView, CellInfo> ();
+
+		class CellInfo {
+			public ICellRenderer Cell;
+			public int Column;
+		}
```

Comment:
I don't think we need a new class for this. This information can be stored in CellViewBackend.